workalendar/africa/kenya.py

```diff
@@ -0,0 +1,70 @@
+from copy import copy
+from datetime import timedelta, date
+
+from ..core import WesternCalendar, IslamicMixin, ChristianMixin
+from ..registry_tools import iso_register
+
+SUN = 6
+
+@iso_register('KE')
+class Kenya(WesternCalendar, IslamicMixin, ChristianMixin):
+    "Kenya"
+    include_good_friday = True
+    include_easter_monday = True
+    # Islamic holidays
+    include_eid_al_fitr = True
+    include_day_of_sacrifice = True
+    shift_sunday_holidays = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5, 1, "Labour Day"),
+        (6, 1, "Madaraka Day"),
+        (10, 20, "Mashujaa Day"),
+        (12, 12, "Jamhuri Day"),
+        (12, 31, "New Years Eve"),
+    )
+
+    def get_fixed_holidays(self, year):
```

Comment:
Added this to dynamically introduce 'Moi Memorial Day' in 2020 and to rename some things based on year.

Reply:
that's the right way to go!